.evergreen/config/test-variants.yml

```diff
@@ -0,0 +1,126 @@
+#
+# Test variants - only run for select platforms worth testing on
+#
+buildvariants:
+  # PHP 8.2: test all topologies on all versions of MongoDB
+  - name: test-debian11-php82-local
+    display_name: "Test: Debian 11, PHP 8.2"
+    run_on: debian11-small
+    expansions:
+      FETCH_BUILD_VARIANT: "build-debian11"
+      FETCH_BUILD_TASK: "build-php-8.2"
+    depends_on:
+      - variant: "build-debian11"
+        name: "build-php-8.2"
+    tasks:
+      - ".standalone .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
+      - ".replicaset .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
+      - ".sharded .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
+      - "test-atlas-connectivity"
+      - ".ocsp !.4.4"
+    display_tasks:
+      - name: "test-ocsp-latest"
+        execution_tasks:
+          - ".ocsp .latest"
```

Comment:
[Display tasks](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files/#display-tasks) group a number of tasks in the "configure patch" view. While this limits us to have to select/unselect all OCSP tests at once without having a choice of enabling only some of them, I believe this is fine. Each OCSP test is an individual task due to the way these are structured.

Reply:
SGTM. I think there's also room for improvement here. If we look at libmongoc, I believe they extracted most of their OCSP logic into [run-ocsp-test.sh](https://github.com/mongodb/mongo-c-driver/blob/master/.evergreen/scripts/run-ocsp-test.sh), which makes the Evergreen configuration more readable. That might allow us to do reduce the complexity of `_template-ocsp.yml` and bring it more in line with other templates where we'd only selecting server versions.